addons/website_event_sale/views/event_event_views.xml

```diff
@@ -5,7 +5,7 @@
         <field name="model">event.event</field>
         <field name="inherit_id" ref="event.view_event_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='company_id']" position="attributes">
+            <xpath expr="//form[1]/sheet[1]/group[1]/group[@name='right_event_details']/field[@name='company_id']" position="attributes">
```

Comment:
if the selector is too specific, this increase the probability of breaking XPATH in customer databases that have customization

2 examples of less specific xpaths:

```xml
//field[@name='company_id'][last()]
//group//field[@name='company_id'] 
```